### ISSUE TITLE
Accessibility: Creating a new notebook by pressing `Enter` after focusing the "OK" button creates 3 notebooks

### ISSUE THREAD

**Issue by personalizedrefrigerator** [laurent22/joplin#10815]
### Operating system

Linux

### Joplin version

3.0.13

### Desktop version info

Joplin 3.0.13 (prod, linux)

Client ID: 24b2a961cccf40fe8448cdc0930b6fce
Sync Version: 3
Profile Version: 47
Keychain Supported: No

Revision: 599cf5b86

Backup: 1.4.1
CodeMirror 6 snippets: 0.0.8
Diff view: 0.1.1
Extra markdown editor settings: 1.1.1
Freehand Drawing: 2.11.0
Work tracker: 0.0.1

### Current behaviour

1. Click the "+" to the right of "NOTEBOOKS"
2. Type a title.
3. Press <kbd>tab</kbd> until "OK" is focused.
4. Press <kbd>enter</kbd>.

Currently, this creates 3 notebooks with the same title.

### Expected behaviour

Only one notebook should be created.
